stable/pipelines/ci/global-section-values.yaml

```diff
@@ -0,0 +1,23 @@
+# CI values for Pipelines
+databaseUpgradeReady: true
+global:
+  versions:
+    pipelines: 1.9.2
```

Comment:
Shouldn't be here 1.10.0?

Reply:
@rimusz This is for CI testing previous versions